src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -324,6 +357,19 @@ public UpdateRequest setEmail(String email) {
     }
 
     /**
+     * Updates the phone number associated with this user.
+     *
+     * @param phone a valid phone number string or null.
+     */
+    public UpdateRequest setPhoneNumber(String phone) {
```

Comment:
Done. Added `Nullable` to appropriate method arguments, and also updated the javadoc of `setPhoneNumber()`.